### PR TITLE
Use pass manager for optimizations

### PR DESCRIPTION
**Pull request by slyubomirsky**
The recent pass manager changes have broken the places where AoT optimizes a model. These changes now use the pass manager for performing optimizations.

### CODE REVIEW

File: aot/aot.py [hunk 1]
```diff
@@ -150,21 +131,19 @@ def __init__(self, mod, tgt) -> None:
...
-        fuse_check(fused_e, self.mod)
-        anf_fused = relay.ir_pass.to_a_normal_form(fused_e, self.mod)
-        fuse_check(anf_fused, self.mod)
-        anf_fused = relay.ir_pass.infer_type(anf_fused, self.mod)
-        fuse_check(anf_fused, self.mod)
-        return anf_fused
+    def optimize(self, expr: Function) -> Function:
+        opts = relay.transform.Sequential([relay.transform.FuseOps(),
+                                           relay.transform.ToANormalForm(),
+                                           relay.transform.InferType()])
```
Comment (MarisaKirisame) [inline aot/aot.py]: sequential do infertype, just remove it

**Comment by MarisaKirisame** [comment]
LGTM except that comment, do you have write privilige? Just merge when you are ready.

**Comment by slyubomirsky** [comment]
I don't have write privs. I will address the comment and you'll have to merge.

File: aot/aot.py (hunk 1 above)
Comment (slyubomirsky) [reply aot/aot.py]: Done

**Comment by MarisaKirisame** [comment]
@jroesch can you get @slyubomirsky write?

**Pull request merged**